packages/ariakit/src/checkbox/__examples__/checkbox-as-button/style.css

```diff
@@ -0,0 +1,36 @@
+.checkbox {
+  @apply
+    flex
+    h-10
+    cursor-pointer
+    items-center
+    justify-center
+    gap-1
+    rounded-lg
+    border-none
+    px-4
+    text-base
+    focus-visible:ariakit-outline
+    /* Base Unchecked */
```

Comment:
Nice! I hadn't thought about comments like these, but they make the styles much easier to read.